Client/Frontend/Browser/Tab Management/TabManager.swift

```diff
@@ -184,6 +184,7 @@ class TabManager: NSObject, FeatureFlaggable, TabManagerProtocol {
         configuration.processPool = WKProcessPool()
         let blockPopups = prefs?.boolForKey(PrefsKeys.KeyBlockPopups) ?? true
         configuration.preferences.javaScriptCanOpenWindowsAutomatically = !blockPopups
+        configuration.mediaTypesRequiringUserActionForPlayback = AutoplayAccessors.getMediaTypesRequiringUserActionForPlayback(prefs);
```

Comment:
Same thing with the `;` at the end

Reply:
you can tell i usually write c++ :-/
done, thanks. you are a human swiftlint.